src/components/TvDetails/index.tsx

```diff
@@ -382,8 +384,12 @@ const TvDetails: React.FC<TvDetailsProps> = ({ tv }) => {
                 <StatusBadge
                   status={data.mediaInfo?.status}
                   inProgress={(data.mediaInfo.downloadStatus ?? []).length > 0}
-                  plexUrl={data.mediaInfo?.plexUrl}
-                  plexUrl4k={data.mediaInfo?.plexUrl4k}
+                  plexUrl={
+                    data.mediaInfo?.plexUrl ?? data.mediaInfo?.jellyfinUrl
+                  }
+                  plexUrl4k={
+                    data.mediaInfo?.plexUrl4k ?? data.mediaInfo?.jellyfinUrl4k
+                  }
```

Comment:
I don't thing this naming is good anymore. Maybe `mediaUrl` and `mediaUrl4k` or send them as seperate props.

Reply:
I think that we can add a `setMediaUrl()` method to `Media`, similar to `setDisplayName()` in `User`.  Then we have flexibility to make changes later if we ever decide we want to support multiple media servers per Overseerr instance.

Reply:
Agreed!

Reply:
It will also allow you to revert the changes to the logic here.  Simply replace `plexUrl` with `mediaUrl`